tasks/update-version.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * A task that updates version numbers in project files in accordance with
+ * build config.
+ *
+ * @param {Object} grunt The grunt helper object.
+ * @returns {void}
+ */
+module.exports = function( grunt ) {
+	grunt.registerMultiTask(
+		"update-version",
+		"Updates the version tags in all the right places.",
+		function() {
+			let options = this.options(
+				{
+					version: "version",
+					regEx: "regEx",
+					preVersionMatch: "preVersionMatch",
+					postVersionMatch: "postVersionMatch"
+				}
+			);
+			// foreach file in this.files
+			for ( let i = 0, n = this.files.length; i < n; i++ ){
+				// foreach src in file
+				for ( let p = 0, n = this.files[ i ].src.length; p < n; p++ ){
```

Comment:
See previous comment. I'd suggest changing the `p` to `j` to better adhere known standards.